modules/ROOT/pages/_partials/sync/server-data-access.adoc

```diff
@@ -1,3 +1,125 @@
 = Data Access
 
-TODO
\ No newline at end of file
+It is very likely a {sync-service} application will need to access data from a database for it to work. The GraphQL spec or Voyager libraries does not require an application to use a particular type of database, and the developer of the application can choose a database that is most suitable for the application. 
+
+From a very high level, here are the steps to access data from a database in a {sync-service} application:
+
+. Initialize the database connection on app start
+. Add the database connection to the `context` object
+. Retrieve/persist data to the database in the application's resolvers
+
+== Initialize Database Connection
+
+Generally speaking, the application should set up the database connection on start, and fail if the connection can not be established. In order to connect to the database, usually the information like the hostname and port number of the database server, the credentials to authenticate etc will be required. It is recommended to use environment variables to set these details. 
+
+If the application is going to be deployed on OpenShift as described in the <<Running {sync-service} on OpenShift>> section, then the following environment variables will be setup for the application automatically:
```

Comment:
so, these env vars aren't available when doing local dev?
Would it be typical that user *doesn't* hook up voyager server to db during local dev times?

Reply:
no, these env vars won't be available locally unless the developer set them explicitly. 

No, that's not typical. Usually devs do hook up to db during local dev.

Reply:
so, i think the text is ambiguous here, what's the best way for me to propose edit, after merging this PR?

Reply:
create a new PR after this is merged?